nc-news-frontend/src/components/ArticleCard.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import moment from 'moment'
+import commentIcon from '../assets/commentIcon.png'
+
+
+
+const ArticleCard = ({title, author, created_at, article_img_url, comment_count}) => {
+  return (
+    <div className='content-card'>
+        <div className='title'> <p>{title}</p></div>
+        <img src={article_img_url} alt="article's image" className='article-img' />
```

Comment:
just a side note - we should not use the word image in an alt tag as it already knows it's an image